app/presenters/courses/curriculum_presenter.rb

```diff
@@ -206,7 +206,7 @@ def users
         .where(id: user_ids)
         .with_attached_avatar
         .map do |user|
-          details = user.attributes.slice('id', 'name', 'title')
+          details = user.attributes.slice('id', 'name','title')
```

Comment:
Why was the space removed? 🤔 
```suggestion
          details = user.attributes.slice('id', 'name', 'title')
```